pkg/timestream/datasource.go

```diff
@@ -234,7 +234,7 @@ func (ds *timestreamDS) CallResource(ctx context.Context, req *backend.CallResou
 		}
 		// TODO: Use API endpoint to list tables
 		v, err := ds.Runner.runQuery(ctx, &timestreamquery.QueryInput{
-			QueryString: aws.String(fmt.Sprintf("SHOW TABLES FROM %s", opts.Database)),
+			QueryString: aws.String(fmt.Sprintf("SHOW TABLES FROM %s", applyQuotesIfNeeded(opts.Database))),
```

Comment:
the same will be needed for the query in the line 255

Reply:
Thanks, missed that